src/smart-components/role/add-role/role-information.js

```diff
@@ -42,7 +42,7 @@ const RoleInformation = (formData, onHandleChange) => {
               id="description"
               name="description"
               value={ description }
-              onChange={ (_, event) => onHandleChange({ description: event.currentTarget.value }) }
+              onChange={ (_, event) => onHandleChange({ description: event.currentTarget.value }, true) }
```

Comment:
Shouldn't this be `event.currentTarget.value.trim().length > 0`?

Reply:
If you mean the second parameter like: `onHandleChange({ description: event.currentTarget.value, event.currentTarget.value.trim().length > 0 }`  than no. Description is not required, so the form is valid even if user doesn't write anything there.

Reply:
Oh, I see. Okay, then it's good to go!